Sources/HipMobileUI/Pages/MainPage.xaml

```diff
@@ -33,7 +33,7 @@
                 <BoxView Grid.Row="0" Style="{DynamicResource BoxView}"/>
                 <Label Grid.Row="0" Text="{helpers:Translate MainPage_MenuTitle}"
                        TextColor="White" FontSize="Small" FontAttributes="Bold"
-                       Margin="16,0,0,32" VerticalOptions="End" />
+                       Margin="16,0,0,32" VerticalOptions="End" Style="{DynamicResource Ssp-Black}" />
```

Comment:
Maybe you forgot to revert the style changes in this file or did you intentionally keep them in to promote the new fonts?

Reply:
I did it on purpose, just want to make sure the style in resourcedisctionary is useable

Reply:
will revert it back after finished all the required changes here, and change them in my next PR

Reply:
As you stated yourself, you should revert the changes in this file.